.storybook/stories/LendByCategoryHomepage.stories.js

```diff
@@ -7,7 +7,57 @@ Vue.use(kivaPlugins)
 import StoryRouter from 'storybook-vue-router';
 import LendByCategoryHomepage from '@/pages/Homepage/LendByCategoryHomepage';
 import apolloStoryMixin from '../mixins/apollo-story-mixin';
+import { returnArrayOfLoans } from '../utils';
 
+const mockedAPIResponse = {
+	data: {
+		general: {
+			homepage_category_rows: {
+				key: "ui.homepage_category_rows",
+				value: "[{\"id\":52},{\"id\":96},{\"id\":93},{\"id\":89},{\"id\":87}]",
+			}
+		},
+		lend: {
+			loanChannelsById:[
```

Comment:
Maybe later we could also create a LoanChannels mock data file, it could return multiple named objects for use in various specific situations.

Reply:
@mcstover I thought about this too, will do a follow up PR